posts/2024/2024_07_26_Inigo_week_9.rst

```diff
@@ -0,0 +1,54 @@
+Week 9 into GSoC 2024: The Conditional VAE implementation
+=========================================================
+
+.. post:: July 26 2024
+    :author: Iñigo Tellaetxe
+    :tags: google
+    :category: gsoc
+
+What I did this week
+~~~~~~~~~~~~~~~~~~~~
+
+This week was a bit shorter than usual because Thursday was a holiday in the Basque Country, and today we had an outdoor activity with my lab mates (we went kayaking to the Urdaibai Biosphere Reserve). Nevertheless, it was full of advances and interesting scientific matters.
+
+As I mentioned in my last blog post, this week I worked on the implementation of the conditional VAE based on `this implementation <https://github.com/QingyuZhao/VAE-for-Regression/tree/master>` of this `paper <https://doi.org/10.1007/978-3-030-32245-8_91>`.
```

Comment:
link does not work. wrong formatting I suppose. see here: https://dipy.github.io/preview-html/dipy.org/pull/55/posts/2024/2024_07_26_Inigo_week_9.html

Reply:
Solved